modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/consensus/block/CoalgebraCommand.scala

```diff
@@ -0,0 +1,16 @@
+package org.tessellation.dag.l1.domain.consensus.block
+
+import org.tessellation.dag.l1.domain.consensus.block.BlockConsensusData.{
+  BlockProposal,
+  CancelledBlockCreationRound,
+  Proposal
+}
+
+sealed trait CoalgebraCommand
```

Comment:
It's only used inside of the Cell, we have `ConsensusData` extending `Ohm` and coming from the outside to the cell, `CoalgebraCommand` which is only living inside of the cell and doesn't need to extend `Ohm` because of that, `AlgebraCommand` which extends `Ohm` only because using `StackF` requires that and `ConsensusOutput` which is an output of the cell and extends `Ohm`.